bot.es6

```diff
@@ -245,75 +334,147 @@ class Bot {
 
   static get SQUAREWEIGHTS() {
```

Comment:
A comment here would be appreciated. I also think it's much easier to read if you sort they keys. Right now it's really hard to tell if you forgot to weight one of the squares.